commit 6e65ce6582a34c2f0892f5c8be018e5114b9eb8c (refs/changes/85/4885/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-09-26T11:57:59-07:00 (1 year ago)
    
    MANTA-3966 MAKO_HTTP_KEEPALIVE_TIMEOUT should be set by default

diff --git a/cmd/manta-init.js b/cmd/manta-init.js
index 651e90b..0911f19 100755
--- a/cmd/manta-init.js
+++ b/cmd/manta-init.js
@@ -575,6 +575,9 @@ async.waterfall([
 		extra.metadata['MANTA_TLS_INSECURE'] = '1';
 		extra.metadata['MUSKIE_MULTI_DC'] = false;
 
+		/* See also MANTA-3966 */
+		extra.metadata['MAKO_HTTP_KEEPALIVE_TIMEOUT'] = 86400;
+
 		// This is filled in when marlin is deployed.
 		extra.metadata['SERVER_COMPUTE_ID_MAPPING'] = {};
 
